src/twisted/web/server.py

```diff
@@ -73,7 +73,7 @@ def _addressToTuple(addr):
 
 
 @implementer(iweb.IRequest)
-class Request(Copyable, BaseRequest, components.Componentized):
+class Request(Copyable, http.Request, components.Componentized):
```

Comment:
I kept the attribute access, as I hope this is not a pefroamnce issue.
It should only be resolved once.